# Sensor Params
sensor_params:
  lidarOdometryRate: 5
  gnssRate: 20

# Outlier Rejection
outlier_params:
  gnssOutlierThreshold: 0.3 # in meters, if jumping more than this, it is considered as absent GNSS, occurs between two GNSS measurements

# Noise Parameters
noise_params:
  ## LiDAR
  poseBetweenNoise: [ 10.0, 10.0, 10.0, 2.0, 2.0, 2.0 ]      #Noise of add between factor -ORDER RPY(rad) - XYZ(meters)
  poseUnaryNoise: [ 1000.0, 1000.0, 10.0, 10.0, 10.0, 10.0 ] # ORDER RPY(rad) - XYZ(meters)
  ## GNSS
  gnssPositionUnaryNoise: 1.0 #1.0e-02 # x, y, z of global position
  gnssHeadingUnaryNoise: 100.0 # x,y,z of heading vector
